site2/website/versions.json

```diff
@@ -1,4 +1,5 @@
 [
+  "2.5.2",
   "2.5.1",
```

Comment:
Please adjust version 2.5.1 to the top first, and then adjust version 2.5.2 to the top, after the search is available